driver/remote/driver.go

```diff
@@ -0,0 +1,86 @@
+package remote
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/docker/buildx/driver"
+	"github.com/docker/buildx/util/progress"
+	"github.com/moby/buildkit/client"
+	"github.com/pkg/errors"
+)
+
+type Driver struct {
+	factory driver.Factory
+	driver.InitConfig
+	*tlsOpts
+	buildkitAddr string
+}
+
+type tlsOpts struct {
+	serverName string
+	caCert     string
+	cert       string
+	key        string
+}
+
+func (d *Driver) Bootstrap(ctx context.Context, l progress.Logger) error {
+	return nil
+}
+
+func (d *Driver) Info(ctx context.Context) (*driver.Info, error) {
+	if d.buildkitAddr == "" {
+		return nil, errors.Errorf("buildkitd addr must not be empty")
+	}
+
+	c, err := d.Client(ctx)
+	if err != nil {
+		return nil, errors.Wrapf(driver.ErrNotConnecting, err.Error())
+	}
+
+	if _, err := c.ListWorkers(ctx); err != nil {
+		return nil, errors.Wrapf(driver.ErrNotConnecting, err.Error())
+	}
+
+	return &driver.Info{
+		Status: driver.Running,
+	}, nil
+}
+
+func (d *Driver) Stop(ctx context.Context, force bool) error {
+	return fmt.Errorf("stop command is not implemented for this driver")
+}
+
+func (d *Driver) Rm(ctx context.Context, force, rmVolume, rmDaemon bool) error {
+	return fmt.Errorf("rm command is not implemented for this driver")
```

Comment:
You still need to be able to remove the remote builder instance. The buildkitd itself remains running.